### ISSUE TITLE
SourceDocs seems to be skipping all open classes?

### ISSUE THREAD

**Issue by abushnaq-work** [eneko/SourceDocs#22]
## Brief Description
I run `sourcedocs generate -c -l -t` in the root of my workspace. I can see all of my classes being parsed but all open classes are being skipped.

## Steps to Reproduce
1) Create a framework or module with classes that are open and classes that are public.
2) Run `sourcedocs generate -c -l -t`.

### Expected Outcome
All classes should have a generated markdown file.

### Actual Outcome
Only classes that have public accessibility are generated. This is because `hasPublicACL` is only checking for public, not open in this class: https://github.com/eneko/SourceDocs/blob/b5bff79481b576eca99f9621296b8f33f72e1e11/Sources/SourceDocs/Classes/SwiftDocDictionary.swift
`

## Environment

- **Operating System:** macOS 10.14.5
- **Xcode Version:** 10.2
- **SwiftDocs Version/SHA:** 0.1.0 -> Not sure.
- **Swift version:** 5.0

## Additional Comments
I tried to build this with `make install` so that I can fix this bug and submit a PR but I ran into linker issues(this is just part of the log it was too long for the issue):

```
 "_swift_storeEnumTagSinglePayloadGeneric", referenced from:
      _$s10Commandant8ArgumentVwst in Argument.swift.o
      _$s10Commandant6OptionVwst in Option.swift.o
      _$s4Yams4Pair33_2EA47F6402A38415204E0D1E326B0671LLVwst in Node.Mapping.swift.o
  "_swift_unexpectedError", referenced from:
      _$sSo8CXCursora21SourceKittenFrameworkE3strSSSgyF in Clang+SourceKitten.swift.o
      _$sSo8CXCursora21SourceKittenFrameworkE4nameSSyFTf4x_n in Clang+SourceKitten.swift.o
      _$s21SourceKittenFramework4FileC8contentsSSvgyyXEfU_ in File.swift.o
      _$s21SourceKittenFramework6ModuleV19xcodeBuildArguments4name6inPathACSgSaySSG_SSSgSStcfC in Module.swift.o
      _$s21SourceKittenFramework0A11DeclarationV11accessorUSR33_D57F9DC6F0FBA1C341F3C3063B352D9ALL6getterSSSb_tF in SourceDeclaration.swift.o
      _$sSS21SourceKittenFrameworkE11commentBody5rangeSSSgSo8_NSRangeVSg_tFTf4nx_n in String+SourceKitten.swift.o
      _$sSS21SourceKittenFrameworkE11pragmaMarks8filename13excludeRanges5limitSayAA0A11DeclarationVGSS_SaySo8_NSRangeVGAJSgtFTf4xnnx_n in String+SourceKitten.swift.o
      ...
  "_swift_unknownObjectRelease", referenced from:
      _$sSD17dictionaryLiteralSDyxq_Gx_q_td_tcfCTf4gn_n in Argument.swift.o
      _$s10Commandant14ArgumentParserCyACSaySSGcfcTf4gn_n in ArgumentParser.swift.o
      _$sSayxSicir in Command.swift.o
      _$s10Commandant15CommandRegistryC4main9arguments11defaultVerb12errorHandlers5NeverOSaySSG_SSyxXEtFTf4nxnn_n in Command.swift.o
      _$s10Commandant11HelpCommandV3runy6ResultAEOyytxGAA0B7OptionsVyxGF in HelpCommand.swift.o
      _$s7Rainbow12OutputTargetO7currentACvpZfiACyXEfU_ in OutputTarget.swift.o
      _$s9SWXMLHash13IndexingErrorOWOe in SWXMLHash.swift.o
      ...
  "_swift_unknownObjectRetain", referenced from:
      _$sSD17dictionaryLiteralSDyxq_Gx_q_td_tcfCTf4gn_n in Argument.swift.o
      _$s10Commandant14ArgumentParserCyACSaySSGcfcTf4gn_n in ArgumentParser.swift.o
      _$sSayxSicir in Command.swift.o
      _$s9SWXMLHash13IndexingErrorO11descriptionSSvg in SWXMLHash.swift.o
      _$ss12_ArrayBufferV19_getElementSlowPathyyXlSiF9SWXMLHash7IndexOpC_Tg5Tm in SWXMLHash.swift.o
      _$s9SWXMLHash13IndexingErrorOWOy in SWXMLHash.swift.o
      _$s9SWXMLHash13IndexingErrorOWOy in XMLIndexer+XMLIndexerDeserializable.swift.o
      ...
  "_swift_updateClassMetadata2", referenced from:
      _$s9SWXMLHash0A7OptionsCMr in SWXMLHash.swift.o
  "_swift_willThrow", referenced from:
      _$s10Commandant14CommandWrapperVyxGADSbs5Error_pIgggdzo_A2DSbsAE_pIegnndzo_sAERzlTRTA in Command.swift.o
      _$s10Commandant14CommandWrapperVyxGSis5Error_pIggdzo_ADSisAE_pIegnrzo_sAERzlTRTA in HelpCommand.swift.o
      _$s10Commandant10OrderedSetVyxGSTAAST32withContiguousStorageIfAvailableyqd__Sgqd__SRy7ElementQzGKXEKlFTW in OrderedSet.swift.o
      _$sxSSSgs5Error_pIgnozo_xAAsAB_pIegnrzo_lTRTA in Extensions.swift.o
      _$s6ResultAAO3getxyKF in Result.swift.o
      _$s9SWXMLHash10XMLIndexerOyACyXlKcfC in SWXMLHash.swift.o
      _$s9SWXMLHash10XMLIndexerO5byKeyyACSSKFTf4xn_n in SWXMLHash.swift.o
      ...
ld: symbol(s) not found for architecture x86_64
```

**Comment by abushnaq-work** [comment]
Hi @eneko 

I have a local fix for this but I'm not sure how to submit a PR without being able to push a remote branch? Sorry if it's a silly question but it's my first open source contribution. :)

**Comment by eneko** [comment]
Hi @abushnaq-work!

First of all, thank you for reporting the issue and thank you for contributing. 

In order to submit a Pull Request, you will need to do the following:
1. At the top right of this page, tap the Fork button.
2. Once you have a clone of SourceDocs in your account/organization, apply the changes to the `master` branch.
3. On the root page of the repository you forked, you will see a message indicating your repository is 1 (or more) commits ahead from `eneko/SourceDocs`, and a button to submit a pull request.

Let me know if that makes sense. Looking forward to that fix.

Thank you!
Eneko

**Comment by minuscorp** [comment]
Duplicate of #19 fixed in #28
